src/clib/grackle_fortran_types.def

```diff
@@ -26,7 +26,7 @@
       integer, parameter :: RKIND=8
 #endif
 
-#define tiny8 1.d-20
+#define tiny8 1.d-40
```

Comment:
I assume there is a reason for this change, but it seems like if we make it to tiny, we should set huge to 1.d+40 to match.

Reply:
My recollection is that the original value of `tiny8` resulted in non-negligible heating of very low density gas when simulating past z = 0. It looks like `tiny8` and `huge8` or only used in a handful of placed, relative to `tiny` and `huge`, so it's probably safe to increase `huge8` to 1d40. I will do that.